site/src/content/docs/best-practices/upgrading-zarf.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: "Upgrading Zarf"
+---
+
+Optimally the version of Zarf performing the `package create` and `package deploy` operations aligns to the version of [Init Package]() that is installed within the cluster.
+
+Follow [CLI Installation](../../getting-started/install) steps for installing updates to the version of binary installed on the machine executing zarf commands.
+
+# Upgrading the Init Package
+
+Upgrading the init package is a bit different from upgrading a regular application package because it establishes your cluster's foundational infrastructure. Follow these steps to upgrade your init package:
+
+## 1. Obtain the Updated Init Package
+
+- **Default Package:** If you're using the default package, run the following command to fetch the latest init package (if it's not already present locally):
+
+  ```bash
+  zarf tools download-init
+  ```
+
+  - **Validate:** This will download the associated init package to your current working directory unless `--output-directory` is specified.
+
+- **Manual:** The Init Package can also be manually downloaded from the Github releases to a storage location of your choice.
+
+  ```bash
+  VERSION="<target version>"; ARCH="<target architecture>"; curl -LO https://github.com/zarf-dev/zarf/releases/download/${VERSION}/zarf-init-${ARCH}-${VERSION}.tar.zst
+  ```
+
+## 2. Re-initialize the cluster
+
+- **Init:** The init process can be executed to upgrade a cluster from one version to another.
+
+  ```bash
+  zarf init --confirm
+  ```
+
+- **Deploy:** The Init Package can also be deployed directly with `package deploy` - allowing for explicit deployment specification.
```

Comment:
I think it's worth mentioning here that custom init packages can be deployed this way

Reply:
agreed!